### ISSUE TITLE
Some questions about training

### ISSUE THREAD

**Issue by logic1988** [mgharbi/hdrnet_legacy#35]
@mgharbi Thank you very much for your great work, very helpful to me. Can you help me for the following questions:

1. why use tf.image.ResizeMethod.NEAREST_NEIGHBOR but not bilinear in https://github.com/mgharbi/hdrnet_legacy/blob/master/hdrnet/data_pipeline.py#L169
2. I used fivek dataset for training, 100 of which were used as eval sets. But why the eval pnsr was so low, only about 8. 
<img width="698" alt="a" src="https://user-images.githubusercontent.com/13864726/48978956-e33b5800-f0ee-11e8-9215-11e8409e97d9.png">
3. Can you share some of the experience in training image enhancement tasks? How to adjust those super parameters, such as learning rate, batchnorm, etc.

**Comment by logic1988** [comment]
I found the reason for point 2: code in https://github.com/google/hdrnet/blob/master/hdrnet/bin/train.py#L105 was wrong.

**Comment by mgharbi** [comment]
1.  We used nearest neighbor to match our fast Android implementation with the Camera2 API
  2.  Don’t hesitate to submit a pull request with your fix.
Best,
Michael
